apis/v1alpha1/generator.yaml

```diff
@@ -90,8 +90,6 @@ resources:
     exceptions:
       terminal_codes:
         - DBClusterQuotaExceededFault
-        - DBParameterGroupNotFound
-        - DBSubnetGroupNotFoundFault
         - DBSubnetGroupDoesNotCoverEnoughAZs
         - DomainNotFoundFault
```

Comment:
the `DomainNotFoundFault` was referring to an existing Active Directory domain. it's the resource inside aws ActiveDirectory service , i suppose it's not able to be created in k8s for now. 
`KMSKeyNotAccessibleFault` is a reasonable one to remove since i see kms-controller and i suppose k8s now can create kms key

Reply:
> i suppose it's not able to be created in k8s for now
I don't think this should be relevant for us either. It's possible that someone has already created a Domain through the console, CloudFormation, CDK, Terraform, etc. They might apply their K8s resources and afterwards create the CloudFormation stack which contains the Domain. We don't expect that everything will be created with ACK, but we should still account for it being created somewhere else

Reply:
makes sense